README.md

```diff
@@ -92,6 +92,27 @@ This just tells Commitizen which adapter we actually want our contributors to us
 
 Please note that in previous version of Commitizen we used czConfig. **czConfig has been deprecated** and you should migrate to the new format before Commitizen 3.0.0.
 
+#### Optional: Install and run Commitizen locally
+
+Installing and running Commitizen locally allows you to make sure that developers are running the exact same version of Commitizen on every machine.
+
+Install Commitizen with `npm i -D commitizen`.
```

Comment:
For consistency I would spell it out `npm install --save-dev commitizen`, since you spelled it out on line 103.